rites/roman1969/build/data-checks.ts

```diff
@@ -20,22 +20,24 @@
  * to distinguish them from the keys that are directly defined in the source files.
  */
 
+/* eslint-disable no-console */
+/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
see other comment about this type of configuration

no-console could be just ignored in an override in a similar manner in `build` folders